src/www/wizard.php

```diff
@@ -172,6 +172,7 @@ function find_ip_interface($ip, $bits = null) {
 function update_config_field($field, $updatetext, $unset, $arraynum, $field_type) {
 	global $config;
 	$field_split = explode("->",$field);
+    $field_conv = '';
```

Comment:
indent in this file is probably still whacky tabs?

Reply:
Yes :'(
What should I do with it? I can open a PR to reformat this file.

Reply:
sure, can do this after merging this one. wizard.php, system.xml and openvpn.xml (all wizard files) still use tabs. thanks!